### DISCUSSION TITLE
Could not find suitable archiver

### DISCUSSION THREAD

**Discussion by aprilnovak** [idaholab/moose#21763] [Q&A Getting Started] [answered]
I am helping a new Cardinal user, @zsh151, get started. They are running into the following issue when trying to build PETSc:

```
Could not find a suitable archiver. Use --with-ar to specify an archiver.
```

<img width="377" alt="182544982-f041d094-df6d-42b3-ac42-3e1502bd5cc8" src="https://user-images.githubusercontent.com/17039662/182643893-302ae663-b894-4333-bd23-b65b1e7e7dad.png">

Would you please be able to provide assistance? I think this issue is probably not specific to Cardinal, hence why I am moving the question from the [Cardinal discussions page](https://github.com/neams-th-coe/cardinal/discussions/445) to here.

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Did you try passing the option mentioned?
Googling it I find results about Thunar and the GNOME fileroller, not sure how helpful that is

@milljm are you familiar with this?

**Reply by zsh151** [reply]
I tried run `--with-ar`, then
<img width="630" alt="image" src="https://user-images.githubusercontent.com/110072634/183041760-4a59dc7a-9c64-43b8-a836-bd65f0d96afa.png">

**Reply by milljm** [reply]
I think they made progress in discussions at Cardinal. I posted there... 

https://github.com/neams-th-coe/cardinal/discussions/445

**Reply by GiudGiud** [reply]
Yeah that part seems solved, they've moved on to a weird error, probably a borked environment at this point
Closing
